client/src/components/common/EditableElement.tsx

```diff
@@ -0,0 +1,75 @@
+import { useState, useEffect } from "react";
+import { FiEdit3, FiSave } from "react-icons/fi";
+
+interface Props {
+  value: string;
+  Element: any;
+  onEdit(temp: string): void;
+  isEditable: boolean;
+  style?: any;
+}
+
+const EditableElement = (props: Props) => {
+  const { value, Element, onEdit, isEditable, style } = props;
+
+  const [temp, setTemp] = useState("");
```

Comment:
Can we have a more specific name for the state? Not able to infer the context from the state name

Reply:
Added more specific name and comment for more clarity